reports.py

```diff
@@ -0,0 +1,153 @@
+from reportlab.lib.pagesizes import letter
+from reportlab.pdfgen import canvas
+from datetime import datetime
+import os
+from database import SqlAccess
+
+
+class Report():
+    def __init__(self, unique_id):
+        self.access = SqlAccess(unique_id)
+
+    @staticmethod
+    def calculate_number_of_pages(data):
+        """
+        Calculating number of pages the report will have
+        """
+        items_per_page = (615 - 40) // 16
+        total_pages = (len(data) + items_per_page - 1) // items_per_page
+        return total_pages + 1
+
+    def create_basic_pdf(self, file_name, company_name, title, start_date, end_date):
```

Comment:
rename this function to something more descriptive of the actual function (like create_all_user_pdf)